src/components/ChangePlan/ChangePlan.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import { useIntl } from 'react-intl';
 import { useRouteMatch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import Card from './Card/Card';
+import CardPrice from './Card/CardPrice';
+import CardAction from './Card/CardAction';
+import Ribbon from './Card/Ribbon';
```

Comment:
Que opinás de agrupar esos componentes?

```suggestion
import { Card, CardPrice, CardAction, Ribbon } from './Card';
```

Reply:
perfecto